.github/workflows/build.yml

```diff
@@ -5,9 +5,8 @@ name: build
 
 on:
   push:
-    branches: [ main, develop ]
   pull_request:
-    branches: [ main, develop ]
+    branches: [ staging, release ]
```

Comment:
For what it’s worth, the upstream repository doesn’t even have a main branch. staging would be our analogue to its develop branch. We won’t ever open a PR targeting the release branch, so that part will never run.

Reply:
`release` came up in the discussion about why temaki sprites weren't being committed so I left it in as a reminder to myself to try and understand the upstream flow.